### PR TITLE
Clean up and expand .gitignore file

### PR DESCRIPTION
**Pull request by rye**
- Adds `/logs` to the `.gitignore`.
- Removes `#`-preceded lines after comments

Anyone have recommendations on stuff I can remove in addition to what I've done here? Worth a pass through just to see if anything here actually hits the ignores.

### CODE REVIEW

**Comment by hawkrives** [comment]
You could remove

```diff
-# BUCK
-buck-out/
-\.buckd/
-android/app/libs
+# Android Signing Keys
*.keystore

-# KEYS
-.env.js
```

**Review by hawkrives — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -23,29 +21,21 @@ DerivedData
...
 node_modules/
 npm-debug.log
 yarn-error.log
 .eslintcache
 
-# BUCK
-buck-out/
-\.buckd/
-android/app/libs
+# Android Signing Keys
```
Comment (hawkrives) [inline .gitignore]: looking at this again, this section could move down and be joined up with the `# android keystores` section

File: .gitignore [hunk 2]
```diff
@@ -64,7 +54,7 @@ scripts/travisci_deploy_key
 fastlane/play-private-key.json
```
Comment (hawkrives) [inline .gitignore]: and this line could also move down to `# android keystores`, and we could probably remove the `travisci_deploy_key` line, too.

File: .gitignore (hunk 1 above)
Comment (rye) [reply .gitignore]: Actually, the `*.keystore` rule is a duplicate.

**Pull request merged**